build.sbt

```diff
@@ -151,6 +151,24 @@ lazy val repl = project
 //    fork in (Test, testOnly) := true
   )
 
+
+/**
+ * REPL available via remote ssh access. Pluggable into any app environment to allow live hacking on a live application.
+ */
+lazy val sshd = project.
+    dependsOn(repl).
+    settings(sharedSettings:_*).
+    settings(
+      name := "ammonite-repl-sshd",
+      libraryDependencies ++= Seq(
+        "org.apache.sshd" % "sshd-core" % "0.14.0",
+        //-- test --//
+        "org.slf4j" % "slf4j-nop" % "1.7.12" % "test",
```

Comment:
Apache sshd depends on `slf4j-api` for logging, which will init default console appender if it will not find anything else at runtime. By adding this dependency this log is redirected to the void so that we have readable console output.